airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/config.py

```diff
@@ -17,6 +17,8 @@
 NIGHTLY_GHA_WORKFLOW_ID = "connector_nightly_builds_dagger.yml"
 CI_TEST_REPORT_PREFIX = "airbyte-ci/connectors/test"
 CI_MASTER_TEST_OUTPUT_REGEX = f".*master.*output.json$"
+ANALYTICS_BUCKET = "ab-analytics-connector-metrics"
+ANALYTICS_FOLDER = "data/connector_quality_metrics"
```

Comment:
wondering if we prefer to env var these for obfuscation

Reply:
I think in this case we can hard code 🤷 

Less chance for people stumbling when setting up for the first time due to yet another config.

We can always change later.

Reply:
Reasonable!